cibuildwheel/macos.py

```diff
@@ -271,22 +271,29 @@ def setup_python(
     # PyPy defaults to 10.7, causing inconsistencies if it's left unset.
     env.setdefault("MACOSX_DEPLOYMENT_TARGET", "10.9")
```

Comment:
should the default deployment target be set to "11.0" if config_is_arm64 ?

Reply:
I don't think it really matters, but we can make sure it's set to 11. I don't think macOS on Apple Silicon does anything different with this set less than 11.

Reply:
Actually, it might be confusing with BEFORE_ALL, which always sets this to 10.9. I think it might be better to leave as is?

Reply:
> I don't think it really matters, but we can make sure it's set to 11. I don't think macOS on Apple Silicon does anything different with this set less than 11.

> Actually, it might be confusing with BEFORE_ALL, which always sets this to 10.9. I think it might be better to leave as is?

You're probably right, it probably doesn't matter much & respects the user override, we can leave it as is.